conf/messages

```diff
@@ -245,7 +245,7 @@ nisp.nirecord.gapsinyourrecord.line1= in paid employment and had low earnings
 nisp.nirecord.gapsinyourrecord.line2= unemployed and not claiming benefit
 nisp.nirecord.gapsinyourrecord.line3= self-employed but did not pay contributions because of small profits
 nisp.nirecord.gapsinyourrecord.line4= living abroad
-nisp.nirecord.gap.viewdetails = View <span class="visuallyhidden">{0} </span>details
+nisp.nirecord.gap.viewdetails = View <span class="visuallyhidden"><abbr title="{0}">{0}</abbr></span>details
```

Comment:
I have removed the "title" attribute  as it is only needed if you have a full date.